tests/mlb/test_mlb.py

```diff
@@ -5,7 +5,7 @@
 from mlbstatsapi.models.teams import Team
 from mlbstatsapi.models.game import Game, Plays, Linescore, BoxScore
 from mlbstatsapi.models.venues import Venue
-from mlbstatsapi.models.sports import Sport
+from mlbstatsapi.models.sports import Sport, sport
```

Comment:
Not sure why you are importing Sport and sport. Why do you import lowercase sport here?